src/Network/Network/ChangeLog.md

```diff
@@ -33,6 +33,7 @@
     - Added deprecation attribute warning to the old cmdlets.
 * Bug fix in ExpressRouteLink MacSecConfig. Added new property `SciState` to `PSExpressRouteLinkMacSecConfig`
 * Updated format list and format table views for Get-AzVirtualNetworkGatewayConnectionIkeSa
+* Updated `set-azExpressRouteGateway` to allow parameter -MinScaleUnits without specifying -MaxScaleUnits
```

Comment:
please move this under "## Upcoming Release"

Reply:
Fixed in the new PR, thank you.